pkg/kube/kube_handler.go

```diff
@@ -136,7 +137,13 @@ func NewClusterConfig(inClusterConfig *rest.Config, userToken string, cluster st
 		return config, nil
 	}
 
-	config.Host = clusterConfig.APIServiceURL
+	if clusterConfig.APIServicePath != "" {
+		//config.Host = path.Join(clusterConfig.APIServiceURL, clusterConfig.APIServicePath)
+		baseURL, _ := url.Parse(clusterConfig.APIServiceURL)
+		config.Host, _ = baseURL.Parse(clusterConfig.APIServicePath)
```

Comment:
@absoludity I wasn't sure which of the 2 approaches here ☝️ were more idiomatic for Golang (i.e. line 141, or lines 142-143). Any opinion?